src/auth/auth.service.ts

```diff
@@ -112,4 +130,60 @@ export class AuthService {
       password: hashedPassword,
     });
   }
+
+  async createQrcodeFor2FA(userId: number): Promise<string> {
+    const email = (await this.userRepository.getUserById(userId)).email;
+    const service = 'Recipe App';
```

Comment:
candidate to be moved to some global consts. Though currently I don't see it being used anywhere else.